bot/exts/code_jams/_creation_utils.py

```diff
@@ -38,7 +38,13 @@ async def _get_category(guild: discord.Guild) -> discord.CategoryChannel:
     Return a code jam category.
 
     If all categories are full or none exist, create a new category.
+    If the main CJ category and the CJ Team's category has the same name
+    it raises a `JamCategoryNameConflictError`
     """
+    main_cj_category = guild.get_channel(Channels.summer_code_jam).name
+    if main_cj_category == CATEGORY_NAME:
+        raise JamCategoryNameConflictError()
+
```

Comment:
This works for creation and is probably good enough for now. However, it would ideally also prevent deletion. There is this method used when a jam ends, which is virtually the same as this method https://github.com/python-discord/sir-robin/blob/2cb8c191a0ac28d98e2f7c9ce0ba035ef9c40a12/bot/exts/code_jams/_cog.py#L186-L188